src/index.js

```diff
@@ -87,12 +83,12 @@ class NanoClamp extends Component {
   }
 
   render () {
-    const {is, text: propText, lines, debounce, ellipsis, ...props} = this.props
```

Comment:
I'm extracting the props of the component for avoid pass it to the `is` component